### ISSUE TITLE
Variant view errors

### ISSUE THREAD

**Issue by abought** [statgen/pheget#46]
Reported by @avsmith . Assigning to @amkwong for triage. Since the feature to query GNOMAD data is pretty new, feel free to bring me in for debugging if needed.

# Description
Certain PheGET variant views are showing incorrectly. 

Anomalies occur when switching between different TSS distance queries:
- Points are bunched to one side of the x-axis (but shouldn't be)
- When points bunched up, there are no x-axis tick labels
- The first view renders fine, but if we cycle back and forth between TSS distance, then later selection of the largest size causes a gnomad API error due to a malformed query.
  - If the requests were truly stateless, we would expect that the later query would work. It's possible that there is a bug in the layout rewriting as we switch between options.

![Screen Shot 2020-01-31 at 1 22 16 PM](https://user-images.githubusercontent.com/2957073/73564020-ba1e0200-442c-11ea-9e52-d1796f0d5ce3.png)

# Steps to reproduce
Search for rs7212 and all the points bunch along the x-axis. Example link: 
https://eqtl.pheweb.org/variant/1_145992816/?minimum_tss_distance=-1000000&maximum_tss_distance=1000000

The view above starts at the largest TSS distance. Click on all the options, then return to the largest. You should see a gnomad error. According to the devtools console, the problem is a malformed query.

**Comment by amkwong** [comment]
We managed to track down both of these errors, and resolved one.

The first error (points bunching together in a single column) was due to our code assuming that all gene names are unique (with unique TSS values), and grouping data points together based on this. The view that showed the error contained multiple genes (with unique ENSG gene IDs) which had the same gene name (RF00003 in our data set; also known as U1), which led to a thrown exception in our sorting code.

For now, we will sort this view based on TSS, which our code will assume to be unique. This part of the issue is assume to be resolved for now.

The second problem lies in gnomad, which accepts short gene names instead of ENSG numbers - this causes problems when the gene name maps to more than 1 ENSG number. This will need to be fixed in gnomad and LocusZoom.
